metaflow/plugins/kubernetes/kubernetes_decorator.py

```diff
@@ -80,6 +76,8 @@ class KubernetesDecorator(StepDecorator):
         memory allocated for this step.
     tmpfs_path: string, optional
         Path to tmpfs mount for this step. Defaults to /metaflow_temp.
+    persistent_volume_claims: Dict[str, str], optional
+        A dictionary of persistent volumes to be mounted to the pod for this step.
```

Comment:
Great addition. Could maybe be more specific in what the values of the dict are, maybe with an example? Something along the lines of
```
example: { "pvc-name": "/path/to/mount/on" }
```

Reply:
yep, good point. 

I've made the docstring more explicit now. Not sure if examples have to follow a particular format. Happy to update if needed. 

Thanks